### PR TITLE
Finished books page buttons functionality

### PR DESCRIPTION
**Pull request by 23nith**
Return to reading list:
chrome_Jer994CtIg

Remove from any list:
chrome_vb82CgR3PM

### CODE REVIEW

File: api/markAsReading.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { authHeaders, baseUrl } from "./base";
+
+export const markAsReading = (
```
Comment (mikodagatan) [inline api/markAsReading.ts]: Do you want to name this `markAsInProgress` instead?

**Review by mikodagatan — APPROVED** [review]
Looks good to me, but please check comments.

File: api/markAsReading.ts (hunk 1 above)
Comment (23nith) [reply api/markAsReading.ts]: Isn't it "mark as read" would mean already finished? Correct me if I am wrong. @mikodagatan

Comment (23nith) [reply api/markAsReading.ts]: > Do you want to name this `markAsInProgress` instead?

Okay will do this instead. Thanks!

**Pull request merged**